php-frontend/src/test/java/org/sonar/php/cache/SymbolTableSerializerTest.java

```diff
@@ -70,6 +72,7 @@ void shouldSerializeAndDeserializeExampleData() {
       List.of(new Parameter("$charset", null, true, false)),
       new FunctionSymbolData.FunctionSymbolProperties(false, false),
       Visibility.PUBLIC,
+      false,
```

Comment:
I think it is time for a builder. Maybe we can create a refactoring ticket.

Reply:
I will create a ticket